apps/src/templates/teacherDashboard/CardContainer.jsx

```diff
@@ -7,7 +7,7 @@ const style = {
   display: 'flex',
   flexDirection: 'row',
   flexWrap: 'wrap',
-  justifyContent: 'space-between'
+  justifyContent: 'flexStart'
```

Comment:
I'm not super familiar with this property - what does it change?

Reply:
It changes how the 5 login type cards are organized in the container. Before there was a container for the non-third party choices ('Secret Words', 'Personal Login', and 'Picture') and a separate container for any third party choices like 'Google Classrooms' or 'Clever'. I combined them since part of the spec was to prioritize third-party choices at the front of the list and when all 5 choices were present in the container it spaced them weird where the top row of 3 looked fine, but the bottom row of 2 had one card all the way to the left and one all the way to the right since the justifyContent told it to space it more evenly with space-between. With flexStart, it just pushes them towards the start so that it orders them properly.

Like [  ] [  ] [  ]    instead of [  ] [  ] [  ]
       [  ] [  ]                           [  ]      [  ]